test_scripts/RC/OnRCStatus/022_notification_by_rc_functionality_disallowed_on_hmi_with_registered_app.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- [SDL_RC] TBD
+--
+-- Description:
+-- In case:
+-- 1) RC functionality is allowed on HMI
+-- 2) RC application is registered
+-- 3) RC functionality is disallowed on HMI
+-- SDL must:
+-- 1) Send OnRCStatus notification with allowed = false to registered mobile application and the HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/OnRCStatus/commonOnRCStatus')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function disableRCFromHMI()
+	common.getHMIconnection():SendNotification("RC.OnRemoteControlSettings", { allowed = false })
+	common.getMobileSession():ExpectNotification("OnRCStatus",
+		{ allowed = false, freeModules = {}, allocatedModules = {} })
+	EXPECT_HMINOTIFICATION("RC.OnRCStatus")
+	:Times(0)
```

Comment:
@GetmanetsIrina 
_RC functionality is disallowed on HMI
SDL must send OnRCStatus notification with allowed = false to registered mobile application and the HMI_

Do we need Times(0) here?